tests/functional/processes/ingest/assert_ingested_records.py

```diff
@@ -12,7 +12,7 @@ def assert_ingested_records(source_name: str) -> list[Record]:
     records = (
         db_manager.session.query(Record)
             .filter(Record.source == source_name)
-            .filter(Record.date_modified > datetime.now(timezone.utc).replace(tzinfo=None) - timedelta(minutes=5))
+            .filter(Record.date_modified > datetime.now(timezone.utc).replace(tzinfo=None) - timedelta(minutes=10))
```

Comment:
hmm - i wouldn't recommend pushing this back to 10 minutes. The records should be ingested fairly quickly. If pushing it back to 10 minutes makes the tests pass, it's likely because it's looking at records pulled in from a previous test run.

Reply:
I'll change the timeout back to 5 minutes and look into fixing the tests failing.